ui/components/MesheryPatterns/MesheryPatternCard.js

```diff
@@ -0,0 +1,267 @@
+//@ts-check
+import React, { useState } from "react";
+import {
+  Avatar, Divider, Grid, IconButton, Typography
+} from "@material-ui/core";
+import Button from "@material-ui/core/Button";
+import DeleteIcon from "@material-ui/icons/Delete";
+import Save from "@material-ui/icons/Save";
+import Fullscreen from "@material-ui/icons/Fullscreen";
+import Moment from "react-moment";
+import FlipCard from "../FlipCard";
+import { makeStyles } from "@material-ui/core/styles";
+import { UnControlled as CodeMirror } from "react-codemirror2";
+import FullscreenExit from "@material-ui/icons/FullscreenExit";
+import TrendingUp from "@material-ui/icons/TrendingUp"
+
+const INITIAL_GRID_SIZE = { xl : 4, md : 6, xs : 12 };
```

Comment:
@asubedy Can we rename this to initialGridSize so that we can maintain a particular naming convention wrt to Camel Case